lib/spack/spack/test/cmd/env.py

```diff
@@ -1921,6 +1921,140 @@ def test_env_include_concrete_remove_env():
     assert test2.path not in lockfile_as_dict["include_concrete"].keys()
 
 
+def configure_reuse_true(env):
```

Comment:
Can you refactor these methods into a single method with an argument for which test it's running? I really don't like the test accessing `globals` with an f-string to figure out which method to call.